tests/ethpm/_utils/test_backend_utils.py

```diff
@@ -45,7 +44,6 @@ def test_get_resolvable_backends_for_supported_uris(uri, backends):
     ),
 )
 @pytest.mark.skipif('WEB3_INFURA_PROJECT_ID' not in os.environ, reason='Infura API key unavailable')
-@pytest.mark.xfail(reason="py-ipfs-http-client library doesn't support go-ipfs v0.7.0")
 def test_get_translatable_backends_for_supported_uris(uri, backends):
```

Comment:
I guess it doesn't make a difference to the test passing, but maybe we want to add the `dummy_ipfs_backend` back in the args here like it [was before](https://github.com/ethereum/web3.py/pull/1768/files#diff-14b4b74dd69848bdda95565da963b1b113afbf93430b9036b57a9a36bb2b574bL48)